Revision en docs (#1323)
mercadopago/devsite-docs: guides/manage-account/reports/account-money/glossary.en.md

| PAYMENT_METHOD_TYPE | Payment method type. <br><br> It can be: <br><br> *credit_card*: credit card.<br>*debit_card*: debit card.<br>*bank_transfer*: transfer.<br>*atm*: ATM. <br>*ticket*: cash.<br>*account_money*: account money. <br><br>|

> | SITE | Country that the Mercado Pago account belongs to: <br><br> MLA: Argentina <br> MLB: Brasil <br> MLM: México <br> MLV: Venezuela <br> MLC: Chile <br> MCO: Colombia <br> MLU: Uruguay <br> MPE: Peru <br><br>|

Reviewer: ```suggestion
| SITE | Country in which Mercado Pago account belongs to: <br><br> MLA: Argentina <br> MLB: Brasil <br> MLM: México <br> MLV: Venezuela <br> MLC: Chile <br> MCO: Colombia <br> MLU: Uruguay <br> MPE: Peru <br><br>|
```
